examples/freertos/Main.cpp

```diff
@@ -0,0 +1,68 @@
+#include "generated/stm32fxxx/PinoutTableDefault.hpp"
+#include "hal_st/instantiations/NucleoUi.hpp"
+#include "hal_st/instantiations/StmEventInfrastructure.hpp"
+#include "osal/Osal.hpp"
+#include "osal/freertos/LowPowerStrategyFreeRtos.hpp"
+#include "osal/freertos_system_time/TimerServiceFreeRtos.hpp"
+#include "services/util/DebugLed.hpp"
+#include <chrono>
+#include <thread>
+
+extern "C" void Default_Handler()
+{
+    hal::InterruptTable::Instance().Invoke(hal::ActiveInterrupt());
+}
+
+main_::NucleoF767ziUi ui;
+infra::Optional<hal::OutputPin> pinGreen;
+
+using namespace std::chrono_literals;
+
+void xPortPendSVHandler();
+void xPortSysTickHandler();
+void vPortSVCHandler();
+
+extern "C" [[gnu::naked]] void SVC_Handler() { asm("b vPortSVCHandler"); }
+extern "C" [[gnu::naked]] void PendSV_Handler() { asm("b xPortPendSVHandler"); }
+extern "C" [[gnu::naked]] void SysTick_Handler() { asm("b xPortSysTickHandler"); };
```

Comment:
can `[[gnu::naked]]` be replaced with `__attribute__((naked))`?

I am surprised `[[gnu::..]]` even works since we set `set(CMAKE_CXX_STANDARD_REQUIRED On)`

Reply:
Even better, __attribute__((naked)) can be replaced by [[gnu::naked]], because the latter is the C++ Standard way :)